wallet/tutorials/javascript-dapp-simple.md

```diff
@@ -0,0 +1,288 @@
+---
+description: Create a simple dapp to integrate with MetaMask.
+toc_max_heading_level: 4
+sidebar_position: 3
+---
+
+# Create a simple dapp 
+
+This tutorial walks you through creating a simple JavaScript dapp and integrating it with MetaMask. 
+It demonstrates the basics of connecting to MetaMask: detecting the MetaMask provider, detecting the user's network, and accessing the user's accounts.
+
+:::caution Learning tutorial
+This tutorial is for educational purposes and connects to MetaMask using the legacy provider object, `window.ethereum`, for the sake of simplicity.
+For deployment in a production environment, we recommend [connecting to MetaMask using EIP-6963](../how-to/connect/index.md) instead. 
+
+[EIP-6963](https://eips.ethereum.org/EIPS/eip-6963) introduces an alternative wallet detection mechanism to the `window.ethereum` provider, and enables dapps to support [wallet interoperability](../concepts/wallet-interoperability.md).
+
+For a full end-to-end tutorial that can be used in production, see the
+[Create a simple React dapp](../tutorials/react-dapp-local-state.md) tutorial.
+:::
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/) version 18+
+
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 9+
+
+- A text editor of your choice, such as [VS Code](https://code.visualstudio.com/).
+
+- [MetaMask](https://metamask.io/) installed in the browser of your choice on your development machine.
+
+## Steps
+
+### 1. Set up a new project
+
+Create a new project using with the following structure:
+
+```text
+simple-dapp/
+├─ src/
+│  ├─ detect.js
+├─ dist/
+│  ├─ index.html
+```
+
```

Comment:
```suggestion
```

Since the new instructions are to use Vite, we don't need this instruction anymore right?